compiler-plugin/src/main/java/io/ballerina/stdlib/http/compiler/HttpDiagnosticCodes.java

```diff
@@ -110,11 +111,16 @@ public enum HttpDiagnosticCodes {
     HTTP_151("HTTP_151", "ambiguous types for parameter '%s' and '%s'. Use annotations to avoid ambiguity", ERROR),
     HTTP_152("HTTP_152", "invalid union type for default payload param: '%s'. Use basic structured anydata types",
             ERROR),
+    HTTP_153("HTTP_153", "cannot initiate interceptors in services via both " +
+            "http:ServiceConfig and http:InterceptableService", ERROR),
 
     HTTP_HINT_101("HTTP_HINT_101", "Payload annotation can be added", INTERNAL),
     HTTP_HINT_102("HTTP_HINT_102", "Header annotation can be added", INTERNAL),
     HTTP_HINT_103("HTTP_HINT_103", "Response content-type can be added", INTERNAL),
-    HTTP_HINT_104("HTTP_HINT_104", "Response cache configuration can be added", INTERNAL);
+    HTTP_HINT_104("HTTP_HINT_104", "Response cache configuration can be added", INTERNAL),
+
+    HTTP_201("HTTP_201", "usage of http:Interceptor|http:Interceptor[] in " +
```

Comment:
```suggestion
    HTTP_201("HTTP_201", "defining interceptor pipeline using `http:ServiceConfig`annotation is deprecated. Mark the service as `http:InterceptableService` instead", WARNING);
```

We have to properly give the alternative option in the warning message. Is it possible to link a sample here?

Reply:
Will check on that and add a link if possible

Reply:
Added a link. However the link is not clickable